Jenkinsfile

```
@@ -17,7 +17,7 @@ pipeline {
         DOCKER = credentials('dockerhub-upboundci')
         AWS = credentials('aws-upbound-bot')
         GITHUB_UPBOUND_BOT = credentials('github-upbound-jenkins')
-        CODECOV_TOKEN = credentials('codecov-stack-gcp')
+        CODECOV_TOKEN = credentials('codecov-provider-gcp')
```

```suggestion
        CODECOV_TOKEN = credentials('codecov-stack-gcp')
```

This should make CI work for you. We can't change the name of this token, but we could create a new token in Jenkins called `codecov-provider-gcp`. However, we need the the actual codecov token, which I am not privy to :sweat_smile:

---

Yeah, I was editing this right as you commented. :) The source of the token is https://codecov.io/gh/crossplane/provider-gcp/settings, though that may only be viewable by org owners.

---

Gotcha :) Do you prefer to update token or just keep using `codevoc-stack-gcp`?